app/AndroidManifest.xml

```diff
@@ -3,7 +3,7 @@
           xmlns:tools="http://schemas.android.com/tools"
           package="org.commcare.dalvik"
           android:versionCode="106"
-          android:versionName="2.37.1">
```

Comment:
Seems like I am missing something obvious though how is it that diff here says versionName as 2.37.1 when it's 2.39 in master currently!

Reply:
...good question

Reply:
This isn't against master, its against 2.37.2 (which they're on). Retargeting for clarity

Reply:
Yeah. Not important. I think it's the conflicts though because of which github diff is getting confused and showing 2.37.1 as the removed version!